notebooks/book2/10/invere_cdf_sampling.ipynb

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
<p>It might be additionally nice to create a GIF like this: <a href="https://upload.wikimedia.org/wikipedia/commons/c/cc/Inverse_Transform_Sampling_Example.gif" target="_blank">Inverse_Transform_Sampling_Example.gif</a> (from Wiki)</p><p> </p><p>It won't go in the book. However, I feel it a good animation to create.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>